node/src/database/db_migrations/migrations/migration_6_to_7.rs

```diff
@@ -0,0 +1,377 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use crate::accountant::big_int_processing::big_int_divider::BigIntDivider;
+use crate::accountant::dao_utils::VigilantRusqliteFlatten;
+use crate::accountant::gwei_to_wei;
+use crate::database::db_migrations::db_migrator::DatabaseMigration;
+use crate::database::db_migrations::migrator_utils::{
+    MigDeclarationUtilities, StatementObject, StatementWithRusqliteParams,
+};
+use crate::sub_lib::neighborhood::RatePack;
+use itertools::Itertools;
+use rusqlite::{Row, ToSql};
+
+#[allow(non_camel_case_types)]
+pub struct Migrate_6_to_7;
+
+#[allow(non_camel_case_types)]
+struct Migrate_6_to_7_carrier<'a> {
```

Comment:
Can you move this down to just before its implementation? It'd be easier to read that way.